README.md

```diff
@@ -473,7 +473,7 @@ Schedules a job to run `name` once immediately.
 `data` is an optional argument that will be passed to the processing function
 under `job.attrs.data`.
 
-Returns the `job`.
+Saves the job instance and returns a promise which resolves with the `job`.
 
 ```js
 agenda.now('do the hokey pokey');
```

Comment:
@mmiller42 : Pls update this line in the documentation as well. We need to add await to this line. 
`const job = await agenda.now('do the hokey pokey');`